src/utils/extractHeaders.ts

```diff
@@ -9,8 +9,12 @@ export function extractHeaders(
   let responseHeaders: ObjectSchema[] = [];
 
   operationGroups.forEach(operationGroup =>
-    operationGroup.operations.forEach(operation =>
-      operation.responses?.forEach(response => {
+    operationGroup.operations.forEach(operation => {
+      const responsesAndExceptions = [
+        ...(operation.responses || []),
```

Comment:
We used to just process success responses. However Error responses can also contain headers